cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-acm-metrics-2ti-object-storage/externalsecret.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: open-cluster-management-observability-acm-metrics-2ti-object-storage
+  namespace: open-cluster-management-observability
+spec:
+  refreshInterval: 15s
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: acm-metrics-2ti-object-storage
+    template:
+      type: Opaque
+      engineVersion: v2
+      data:
+        acm-metrics.yaml: |
+          type: s3
+          config:
+            bucket: "{{ .bucket }}"
+            endpoint: "{{ .endpoint }}"
+            insecure: false
+            access_key: "{{ .access_key }}"
+            secret_key: "{{ .secret_key }}"
+            http_config:
+              tls_config:
+                ca_file: "/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+  data:
+  - secretKey: bucket
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage    # the path-details will be replaced in the overlay patch
+      property: bucket
+  - secretKey: endpoint
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
+      property: endpoint
+  - secretKey: access_key
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
+      property: access_key
+  - secretKey: secret_key
+    remoteRef:
+      key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
+      property: secret_key
```

I meant something like:

```suggestion
    dataFrom:
      - extract:
          key: $ENV/$CLUSTER/open-cluster-management-observability/acm-metrics-2ti/object-storage
```

---

Oh, but you're doing templating, so maybe that won't work. Let me try it out.

---

@schwesig take a look in the `default` namespace of `nerc-ocp-infra`; look at the externalsecret `lars-example-secret` and the corresponding secret `lars-example-secret`.

---

I would love that solution.